util/MySqlMigrations/Migrations/DatabaseContextModelSnapshot.cs

```diff
@@ -533,6 +533,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .HasColumnType("varchar(50)");
 
                     b.HasKey("Id")
+                        .HasName("PK_Grant")
```

Comment:
@withinfocus Any idea why my migrations are adding this?

Reply:
I do not. Unsure why my changes didn't already do this but my guess is it has something to do with Pomelo's workaround with key modifications not immediately representing where the named PK is.